micromasters/settings.py

```diff
@@ -482,7 +482,7 @@
 CELERY_BEAT_SCHEDULE = {
     'batch-update-user-data-every-6-hrs': {
```

Comment:
Think we should update the task name to match the updated schedule.

```suggestion
    'batch-update-user-data-every-friday-every-6-hrs': {
```